packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -102,8 +102,8 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 			__experimentalReusableBlocks: reusableBlocks,
 			__experimentalFetchLinkSuggestions: ( search, searchOptions ) =>
 				fetchLinkSuggestions( search, searchOptions, settings ),
-			__experimentalFetchRemoteUrlData: ( url ) =>
-				fetchRemoteUrlData( url ),
+			__experimentalFetchRemoteUrlData: ( url, options ) =>
+				fetchRemoteUrlData( url, options ),
```

Comment:
Nitpicking: we can probably just do this?

```suggestion
			__experimentalFetchRemoteUrlData: fetchRemoteUrlData,
```